students/navdeep/session11/comprehensions.py

```diff
@@ -0,0 +1,21 @@
+import pandas as pd
+
+music = pd.read_csv("featuresdf.csv")
+
+print(music.head())
+print(music.describe())
+
+def zip_music():
+	dance_zip = [(artists, name, danceability, loudness) for (artists, name, danceability, loudness) in zip(music.artists, music.name, music.danceability, music.loudness) if danceability > 0.8 and loudness < -5.0]
+	#print(dance_zip)
+	return dance_zip
+
+def sort_zip():
+	music_zip = zip_music()
+	music_zip_sorted = sorted(music_zip, key = lambda x: x[0])
```

Comment:
I think you want to change this line to:
`music_zip_sorted = sorted(music_zip, key = lambda x: x[2], reverse=True)`
Otherwise it just sorts the names in an ascending order.